### PR TITLE
#146 Reduce columns for the movie lists of the user home page

### LINKED ISSUE #146: Reduce columns in the lists of movies for User

### PR DESCRIPTION
**Pull request by daweedek2**
Closes #146

### CODE REVIEW

File: src/main/java/kostka/moviecatalog/service/CacheService.java [hunk 1]
```diff
@@ -37,4 +39,21 @@ public CacheService(final RedisService redisService) {
         }
         return Collections.emptyList();
     }
+
+    public List<MovieListDto> getMovieDtosFromMoviesWithKey(final String key) {
```
Comment (crlikcngroup) [inline src/main/java/kostka/moviecatalog/service/CacheService.java:43]: Here it is kinda confusing for me. Now you have MovieListDto, MovieFormDto, this method is called getMovieDto (did you mean MovieListDto? Personally I would not include return type in the method name)

File: src/test/java/kostka/moviecatalog/service/Generator.java [hunk 2]
```diff
@@ -17,8 +17,8 @@ public Movie createMovieWithName(final String name) {
         return movie;
     }
 
-    public MovieDto createValidMovieDto(final String name) {
-        MovieDto dto = new MovieDto();
+    public MovieFormDto createValidMovieDto(final String name) {
```
Comment (crlikcngroup) [inline src/test/java/kostka/moviecatalog/service/Generator.java:20]: And same as above. Method called createValidMovieDto returns MovieFormDto...

**Comment by crlikcngroup** [comment]
Except that naming that confused me a little, it looks OK 👍

File: src/main/java/kostka/moviecatalog/service/CacheService.java (hunk 1 above)
Comment (daweedek2) [reply src/main/java/kostka/moviecatalog/service/CacheService.java]: I have tried to rename the methods a little bit, can you check again please?

**Comment by daweedek2** [comment]
#158 issue is also done by this pull request. The MovieListDto is stored in Redis instead of Movie entity.

**Pull request merged**